docs/defi/flow-yield-vaults/cross-chain.md

```diff
@@ -0,0 +1,315 @@
+---
+title: Cross-Chain Integration
+sidebar_position: 8
+---
+
+# Cross-Chain Integration
+
+FYV enables access to yield opportunities across multiple blockchains through Flow's EVM bridge. This document explains how cross-chain integration works and how to access Ethereum-compatible yield vaults from Flow.
+
+## Flow-EVM Bridge Overview
+
+Flow's EVM bridge connects Flow's Cadence environment with Ethereum Virtual Machine (EVM) compatible chains, enabling seamless asset transfer and smart contract interaction across ecosystems.
+
+### Bridge Architecture
+
+The bridge consists of several components:
+
+**FlowEVMBridgeConfig**: Manages token escrow and minting configuration, maintains token pair mappings (Flow ↔ EVM), and handles bridge fees and limits.
+
+**CadenceOwnedAccounts (COA)**: Enables Cadence contracts to control EVM addresses, allows Cadence to send EVM transactions, and bridges resource-oriented (Cadence) with account-based (EVM) models.
+
+**Token Escrow**: Locks Flow native tokens when bridging to EVM, mints equivalent bridged tokens on EVM side, and maintains 1:1 backing ratio.
+
+### How Bridging Works
+
+**Flow → EVM (Bridging Out):**
+```
+1. User deposits USDC (Flow native) to bridge contract
+2. Bridge locks USDC in escrow vault
+3. Bridge mints equivalent bridged USDC on EVM side
+4. EVM contract receives bridged USDC
+5. Transaction hash recorded for verification
+```
+
+**EVM → Flow (Bridging Back):**
+```
+1. EVM contract burns bridged USDC
+2. Bridge receives burn notification
+3. Bridge unlocks equivalent USDC from escrow
+4. User receives USDC on Flow
+5. Cross-chain operation complete
+```
+
+## mUSDCStrategy: Cross-Chain Yield Farming
+
+The mUSDCStrategy leverages the EVM bridge to access Ethereum-compatible ERC4626 yield vaults.
```

Comment:
Ahh, it's much clearer here than the previous reference I commented on. It's just using the Cross-VM bridge, the other doc I already left a comment on made it seem like it was going to a different chain that was EVM. I would change these references to Cross-VM instead of cross-chain if I'm understanding this piece correctly.